media/videostitcher/create_cdn_key.go

```diff
@@ -25,15 +25,16 @@ import (
 )
 
 // createCdnKey creates a CDN key. A CDN key is used to retrieve protected media.
-// If akamaiTokenKey != "", then this is an Akamai CDN key, or else this is a
-// Cloud CDN key.
-func createCdnKey(w io.Writer, projectID, cdnKeyID, hostname, gcdnKeyname, gcdnPrivateKey, akamaiTokenKey string) error {
+// If isMediaCdn is true, create a Media CDN key. If false, create a Cloud
+// CDN key. To create a privateKey value for Media CDN, see
+// https://cloud.google.com/video-stitcher/docs/how-to/managing-cdn-keys#create-private-key-media-cdn.
+func createCdnKey(w io.Writer, projectID, cdnKeyID, hostname, keyName, privateKey string, isMediaCdn bool) error {
```

Comment:
`isMediaCdn` should not be an argument; sample args should be limited to what's needed for testing https://googlecloudplatform.github.io/samples-style-guide/#minimal-arguments

Also, `CDN` rather than `Cdn` throughout in variable names. https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Changed Cdn to CDN in variable names.

isMediaCdn is needed for testing. This code sample can create two types of keys: Cloud CDN keys or Media CDN keys. This matches the samples in the other languages.